variables.tf

```diff
@@ -1,9 +1,32 @@
-##############################################################################
+########################################################################################################################
 # Input Variables
-##############################################################################
+########################################################################################################################
 
-#variable "my_variable" {
-#  type        = string
-#  description = "A description of my variable"
-#  default     = "default_value"
-#}
+variable "enterprise_crn" {
+  type        = string
+  description = "The CRN of the parent Enterprise account to use."
+}
+
+variable "enterprise_primary_contact_iam_id" {
+  type        = string
+  description = "The IAM id of the parent Enterprise account owner."
+}
+
+
+variable "enterprise_accounts" {
+  description = "List of enterprise child accounts in the enterprise"
+  type        = list(object({ name = string, key_name = string, parent = optional(string, null), owner_iam_id = optional(string, null) }))
```

Comment:
Might make more sense to user if the key "parent" was "parent_key_name" so that they know the value should be another key_name, not a CRN or ID?

Also we could expand the description to explain how these variables should be used (what they keys mean).

Applies to the account_group input as well.